### DISCUSSION TITLE
软件进不去，一直显示Loading?

### DISCUSSION THREAD

**Discussion by osrich** [shuax/MouseInc.Settings#351] [Q&A]
从前几天起一直进不去了，如图所示。删除原来文件，再解压出来，还是进不去。

![image](https://user-images.githubusercontent.com/55370127/187815452-55535690-d94b-49fe-854d-4a205480fc12.png)

**Comment by sumeco** [comment]
你是不是电信网络？

**Reply by osrich** [reply]
是电信啊。
